server/meca/file-generators/manifest.xml

```diff
@@ -32,4 +32,5 @@
         <instance media-type="text/html" href="cover_letter.html"/>
     </item>
 
+    {supplementaryFiles}
```

Comment:
This is now a template rather than an xml file, we should indicate that, either by putting this within a templates folder, or using a templating pattern. We can also follow the same pattern as the [`article.js` file](https://github.com/elifesciences/elife-xpub/blob/develop/server/meca/file-generators/article.js)

Reply:
Agreed - something for the future